src/igvfd/audit/analysis_set.py

```diff
@@ -176,3 +176,43 @@ def audit_analysis_set_files_missing_analysis_step_version(value, system):
             f'links to file(s) {files_with_missing_asv} that are missing `analysis_step_version`.'
         )
         yield AuditFailure(audit_message_missing_step_version.get('audit_category', ''), f'{detail} {audit_message_missing_step_version.get("audit_description", "")}', level=audit_message_missing_step_version.get('audit_level', ''))
+
+
+@audit_checker('AnalysisSet', frame='object')
+def audit_analysis_set_multiple_workflows(value, system):
+    '''
+    [
+        {
+            "audit_description": "Analysis set contains more than one workflows.",
+            "audit_category": "multiple workflows",
```

Comment:
can you rename the category to "unexpected workflow" to be consistent with our naming conventions for audit categories?